src/crate/client/sqlalchemy/dialect.py

```diff
@@ -90,7 +90,8 @@ def process(value):
             if not value:
                 return
             try:
-                return datetime.utcfromtimestamp(value / 1e3).date()
+                return datetime.fromtimestamp(value / 1e3, tz=timezone.utc) \
+                    .replace(tzinfo=None).date()
```

Comment:
If we throw away the tzinfo why not keep using utcfromtimestamp?

Reply:
I think the idea is to be a preparation step to actually allow the user to pass a tzinfo here, later on?